keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/KindController.kt

```diff
@@ -10,9 +10,9 @@ import org.springframework.web.bind.annotation.RestController
 @RestController
 @RequestMapping(path = ["/kinds"])
 class KindController(
-  val plugins: List<ResourceHandler<*, *>>
+  val kinds: List<SupportedKind<*>>
 ) {
   @GetMapping(produces = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE])
   fun get(): Set<String> =
-    plugins.map { it.supportedKind.kind.toString() }.toSet()
+    kinds.map { it.kind.toString() }.toSet()
 }
```

Comment:
This is an example of using the kinds directly from Spring rather than having to look at the handlers.